CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
 # Changelog
 
 - [Changelog](#changelog)
+- [Changelog](#changelog-1)
```

Comment:
Oh no, what happened here :fearful: 
I think there should be only one? I'm confused.